src/components/Button/Button.tsx

```diff
@@ -160,7 +161,9 @@ class Button extends UIComponent<Extendable<IButtonProps>, IButtonState> {
       >
         {hasChildren && children}
         {!hasChildren && iconPosition !== 'after' && this.renderIcon(variables, styles)}
-        {!hasChildren && content && <span className={classes.content}>{content}</span>}
+        {!hasChildren &&
+          content &&
+          Slot.create(content, { defaultProps: { as: 'span', className: classes.content } })}
```

Comment:
Short circuit checks not required for .create() methods.  The Slot will return nothing if content does not exist.

Reply:
removed check